scripts/deploy.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python3
+
+from brownie import Multisig, accounts, network
```

Comment:
As a note, Brownie is deprecated and is no longer being maintained. The successor of Brownie is Ape, using Ape is best practice going forward. https://snakecharmers.ethereum.org/intro-to-ape/